backend/flindt/feedback/views.py

```diff
@@ -132,10 +132,10 @@ def get(self, request):
                 }
         """
         # Get all the roles the user is part off.
-        roles = Role.objects.filter(users__in=[self.request.user.pk], archived=False)
-
-        # When no roles are found return a 404.
-        if not roles:
+        try:
+            roles = Role.objects.filter(users__in=[self.request.user.pk], archived=False)
+        except Role.DoesNotExsist:
```

Comment:
This will not work, because `filter` does not raise DoesNotExist, like `get`does.